src/hqdn3d.cpp

```diff
@@ -20,48 +20,48 @@
     Foundation, Inc., 59 Temple Place, Suite 330, Boston, MA  02111-1307  USA
 */
 
+#include <cmath>
+#include <cstdlib>
 #include <cstdint>
 #include <algorithm>
 
-#include <vapoursynth/VapourSynth.h>
-#include <vapoursynth/VSHelper.h>
+#include <VapourSynth.h>
+#include <VSHelper.h>
 
 typedef struct Hqdn3dData {
     VSNodeRef *clip;
     const VSVideoInfo *vi;
 
-    double lumSpac   = -1.0;
-    double chromSpac = -1.0;
-    double lumTmp    = -1.0;
-    double chromTmp  = -1.0;
-    int restartLap   = -1;
+    double lumSpac;
```

Comment:
Using default values is the preferred solution, as this ensures, that none of those values could be used in incorrect manner, even by accident.

Reply:
It's pointless to give the filter parameters default values here because they will be overwritten by the propGet* functions.